monai/transforms/spatial/array.py

```diff
@@ -3277,8 +3278,10 @@ def filter_count(self, image_np: NdarrayOrTensor, locations: np.ndarray) -> tupl
                 idx = argsort(image_np.sum(tuple(range(1, n_dims))))
             elif self.sort_fn == GridPatchSort.MAX:
                 idx = argsort(-image_np.sum(tuple(range(1, n_dims))))
+            elif self.sort_fn == GridPatchSort.RANDOM:
```

Comment:
Hi @jmnolte, thanks for your quick update!
If you rewrite the `filter_count` method here, it might be a little weird. Theoretically we don't want the `GridPatch` to support random operations, so based on @drbeh  [comment](https://github.com/Project-MONAI/MONAI/pull/6701#pullrequestreview-1522897921), you should probably override this method inside `RandGridPatch`.
Free feel to ask if you have any question.
Thanks!